src/internal_events/generator.rs

```diff
@@ -0,0 +1,18 @@
+use super::InternalEvent;
```

Comment:
Should we split the generator changes out?

Reply:
Can do, although since this PR currently only observes the `events_processed` types and the least opinionated way to produce them is via the generator source, I think folks might benefit from this being included to execute the test subscription noted in the PR.

Either way is cool by me though.

Reply:
I vote leave it in.